emgapi/models.py

```diff
@@ -1224,9 +1260,9 @@ def available(self, request=None):
         """
         query_filters = {
             "all": [
-                Q(study__is_public=1),
-                ~Q(sample__is_public=Sample.SUPPRESSED),
-                Q(run__status_id=Status.PUBLIC) | Q(assembly__status_id=Status.PUBLIC),
+                Q(study__is_private=False),
+                ~Q(sample__is_suppressed=True),
```

Comment:
```suggestion
                Q(sample__is_suppressed=False),
```

Reply:
ah, yeah... mindless search/replace.. I'll change it